.github/renovate.json5

```diff
@@ -3,31 +3,51 @@
   extends: ["config:base"],
   labels: ["dependencies"],  // For convenient searching in GitHub
   pip_requirements: {
-    fileMatch: ["(^|/)requirements([\\w-]*)\\.txt$"]
+    // Match every requirements file except requirements-dev.txt, which is handled with a regex manager below.
+    fileMatch: ["(^|\\/)requirements(-(d|[^d].*|de|d[^e].*|de[^v].*|dev\\w+|))?\\.txt$"]
```

Comment:
this looks curious, is there a particular reason for this regex structure? Would something like `^(?!requirements-dev\.txt)requirements(-\S+)?\.txt$` work?

Reply:
Oh I see. Maybe it's possible to use `fileMatch` in conjunction with [ignorePaths](https://docs.renovatebot.com/configuration-options/#ignorepaths) to explicitly exclude requirements-dev.txt?

Reply:
Good thought! I thought I'd tried that yesterday before creating this monster of a regex, but plugging it in now didn't make the validator yell at me so I'll get back to you on it.